component-interaction/src/app/components/user-details.component/user-details.component.html

```diff
@@ -1,11 +1,11 @@
-<div class="green-background" *ngIf="user.dateOfBirth | isAdult">
+<div [ngClass]="{adult: user.dateOfBirth | userAge | isAdult}">
     <h4>{{user.name}}</h4>
     <p>{{user.email}}</p>
     <p>{{user.dateOfBirth | userAge}}</p>
     <br>
 </div>
 
-<div class="grey-background" *ngIf="!(user.dateOfBirth | isAdult)">
+<div [ngClass]="{minor: !(user.dateOfBirth | userAge | isAdult)}">
     <h4>{{user.name}}</h4>
     <p>{{user.email}}</p>
     <p>{{user.dateOfBirth | userAge}}</p>
```

Comment:
you can have only adult css class in order to avoid duplicate code
delete the second div